### ISSUE TITLE
 Issues setting up sandbox on Windows 10 - and discovered a fix

### ISSUE THREAD

**Issue by atrefonas** [algorand/sandbox#38]
Hello, so I had some issues setting up sandbox on Windows 10 with docker and git bash. Went through the steps exactly. There is a strange error about “the input device is not a TTY” which is related to git bash. The symptoms are that you can’t see much of the output text from sandbox. However, I solved the issue with a one line change in the sandbox code. See these screenshots

It's basically this problem - seems git bash and docker can cause issues without adding 'winpty' in front of commands https://stackoverflow.com/questions/48623005/docker-error-the-input-device-is-not-a-tty-if-you-are-using-mintty-try-prefi/49965690

![image](https://user-images.githubusercontent.com/72869083/112862030-5c626580-9083-11eb-86c1-43a9d329dd28.png)
![image](https://user-images.githubusercontent.com/72869083/112862063-61271980-9083-11eb-9622-64ffbc104f42.png)

**Comment by winder** [comment]
@atrefonas does this work as well?

```
WINDOWS_PREFIX=""
if [ "$(expr substr $(uname -s) 1 10)" == "MINGW32_NT" ] || \
     [ "$(expr substr $(uname -s) 1 10)" == "MINGW64_NT" ]; then
  WINDOWS_PREFIX="winpty "
fi

$WINDOWS_PREFIX docker-compose -f "$SANDBOX_DIR/docker-compose.yml" "$@"
```

**Comment by atrefonas** [comment]
Yes, this worked perfectly. Reproduced the issue and then tried it with the fix. Thank you so much!

**Comment by tylercubell** [comment]
It might help to avoid future issues by suggesting WSL2 on Windows instead of Git Bash. I ran into this issue and another syncing issue (not sure of cause) using the Git Bash installation method. When I switched to WSL2, everything worked as expected.

**Comment by NoelJacob** [comment]
@tylercubell can you tell how you used wsl?

**Comment by NoelJacob** [comment]
I use git busybox on windows. I don't know if it's that but I get:
```
C:\Users\HP\Desktop\sandbox>busybox bash ./sandbox up
./sandbox: line 399: syntax error: unexpected word (expecting ")")
```

**Comment by Silventino** [comment]
This should be merged
